Putting a manually typed version in setup.cfg, as it wouldnt resolve automatically.
Author: Denis Kristak (Inuits)diff -ruN tenacity-8.2.3_orig/setup.cfg tenacity-8.2.3/setup.cfg
--- tenacity-8.2.3_orig/setup.cfg	2023-08-14 15:22:48.000000000 +0200
+++ tenacity-8.2.3/setup.cfg	2023-08-15 19:47:02.512360441 +0200
@@ -1,6 +1,7 @@
 [metadata]
 name = tenacity
 license = Apache 2.0
+version = 8.2.3
 url = https://github.com/jd/tenacity
 summary = Retry code until it succeeds
 long_description = Tenacity is a general-purpose retrying library to simplify the task of adding retry behavior to just about anything.
